source/InputController.cpp

```diff
@@ -92,11 +60,50 @@ bool InputController::init() {
 	});
 #endif
 	success = success && Input::activate<TextInput>();
```

Comment:
If we're no longer using textfields anywhere, we should probably get rid of the adjustments needed to add them in. Essentially, we no longer need to activate/deactivate keyboard for mobile devices and no longer need to activate textinput as well.